modules/extension/wms/src/test/java/org/geotools/data/wms/test/GetStyleOnlineTest.java

```diff
@@ -0,0 +1,136 @@
+/*
+ *    GeoTools - The Open Source Java GIS Toolkit
+ *    http://geotools.org
+ *
+ *    (C) 2004-2008, Open Source Geospatial Foundation (OSGeo)
+ *    (C) 2005, David Zwiers
```

Comment:
This is a new file created this year? So the info should be (C) 2015 Open Source Geospatial Foundation (OSGeo)